platform/linux-generic/Makefile.am

```diff
@@ -175,6 +175,7 @@ noinst_HEADERS = \
 		  include/odp_name_table_internal.h \
 		  include/odp_packet_internal.h \
 		  include/odp_packet_io_internal.h \
+		  include/odp_packet_io_pool.h \
```

Comment:
Should we rename it to odp_packet_io_shm.h and odp_packet_io_shm.c?

Reply:
This file is for allocating the shmem for storing the driver's global data.

The name 'pool' seems to be creating confusion with the packet/buffer pool. May be we should change the name of '_odp_ishm_pool_create/odpdrv_shm_pool_create'.